model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/validation/DeviceTokenValidator.java

```diff
@@ -56,6 +56,12 @@
      */
     private static final Pattern ADM_DEVICE_TOKEN = Pattern.compile("(?i)[0-9a-z\\-_.]{100,}");
 
+    /**
+     * Pattern for Firefox is harder to define that is why we kept it lenient it is at least 100 characters long and can
+     * consist of digits, alphas, - , _ and . and all have one of these separators
+     */
+    private static final Pattern FIREFOX_DEVICE_TOKEN = Pattern.compile("(?i)[0-9a-z\\-_.]{100,}");
```

Comment:
can we have some tests in the InstallationsDAO test? with Firefox installations ?